test/test_enclave_wrangler/test_objects_api.py

```diff
@@ -211,3 +233,206 @@ def test_get_concept_set_version_members(self):
         obj_id = 563193300
         data = get_concept_set_version_members(obj_id, return_detail='full')
         self.assertGreater(len(data), 0)
+
+
+# TODO: new tests by claude to try
+# from pprint import pprint as pp
+# todo: #1: Wherever I put '#1', consider removing the mock and call the actual function
+class TestObjectsApiMocks(DbRefreshTestWrapper):
+    """Tests that specifically use mock APIs
+
+    Scaffolded by: https://claude.ai/chat/7f64ea42-7ab2-43c6-9a86-382bdafc26bf
+    """
+    func_cached_return_map = {
+        # test_get_object_types()
+        get_object_types: os.path.join(TEST_INPUT_DIR, 'test_get_object_types', 'object_types.json')
+    }
+
+    def setUp(self):
+        """Set up mock connection"""
+        self.mock_connection = MagicMock()
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
## Set up: Storing returns of functions we are mocking
At the beginning of any tests using this class, sees if the function being mocked has a cached version of its outputs saved. If not, it fetches and saves in the test input dir.